src/modules/addList.js

```diff
@@ -0,0 +1,144 @@
+import { trim } from 'lodash';
+
+export const form = document.querySelector('form');
+export const newTaskEl = document.getElementById('new-task');
+export const toDoListEl = document.querySelector('.toDoList');
+export const btnTask = document.querySelector('.btn-task');
+export const clearAll = document.querySelector('.clear-all');
+export const inputEl = document.querySelectorAll('.inputEl');
+let editId;
+let isEdited = false;
+// eslint-disable-next-line import/no-mutable-exports
```

Comment:
Please do not disable eslint to enable best practices of JavaScript.

Reply:
i have recieved your constructive feedback. 😄